.github/workflows/unit.yml

```diff
@@ -2,6 +2,7 @@ name: Unit Test
 
 on:
   push:
+pull_request:  pull_request:
```

Comment:
wrong indentation

Reply:
Not sure what happened there. Seems like I "yanked" twice without noticing.
Fixing it the "way I understood it", as in the other two ymls.

Apart from that, I see I'm getting double runs for functests now (push + pull_request).
Makes sense, but that's not really desirable,
I'll remove the "push" label and only trigger functests (et al) for pull_request

Reply:
Also, if the following:
```  pull_request:
    branches:
      - develop
```
means we only run the compliance test for develop (PRs aiming at develop I must understand that), then, why not remove that restriction?
The compliance test takes like a minute and it doesn't hurt.